LDM-533.tex

```diff
@@ -231,7 +236,17 @@ \subsection{Suspension criteria and resumption requirements}
 
 \subsection{Naming convention}
 
-All tests are named according to the pattern \textsc{prod-xx-yy} where:
+With the introduction of the JIRA Test Management, the following definition have to be considered:
```

Comment:
"Jira", not "JIRA". "Definitions", not "definition".  You don't need the definite article.

It might be helpful to the reader to provide a link (even just to the Confluence page) which says something about what Jira Test Management actually is.